FormlyMapper/FormlyField.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Linio\DynamicFormBundle\FormlyMapper;
+
+abstract class FormlyField implements FormlyFieldInterface
+{
+    /**
+     * @var array
+     */
+    protected $fieldConfiguration;
+
+    /**
+     * @var array
+     */
+    protected $formlyFieldConfiguration;
+
+    /**
+     * @param array $fieldConfiguration
+     */
+    public function setFieldConfiguration(array $fieldConfiguration)
+    {
+        $this->fieldConfiguration = $fieldConfiguration;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFormlyFieldConfiguration()
+    {
+        $this->formlyFieldConfiguration = [];
```

Comment:
Is there a reason you can't just set `$formlyFieldConfiguration` above to have the default value of `[]`?